priv/repo/migrations/20200413202224_add_coturn_schema.exs

```diff
@@ -17,35 +15,6 @@ defmodule Ret.Repo.Migrations.AddCoturnSchema do
     execute(
       "create table coturn.denied_peer_ip (realm varchar(127), ip_range varchar(256), inserted_at timestamp, updated_at timestamp)"
     )
-
-    if coturn_password && coturn_password != "" do
-      execute("""
-      DO
-      $do$
-      BEGIN
-         IF NOT EXISTS (
-            SELECT                       -- SELECT list can stay empty for this
-            FROM   pg_catalog.pg_roles
-            WHERE  rolname = 'coturn') THEN
-
-            CREATE ROLE coturn LOGIN PASSWORD '#{coturn_password}';
-         END IF;
-      END
-      $do$;
-      """)
-
-      execute("""
-      do 
-      $$ 
-      begin
-        execute format('grant connect on database %I to coturn', current_database());
-      end;
-      $$;
-      """)
-
-      execute("grant usage on schema coturn to coturn;")
-      execute("grant all privileges on all tables in schema coturn to coturn;")
```

Comment:
Don't we still have to grant access to the coturn schema to some user?

Reply:
no - the relevant user is now just the superuser `postgres` - we have the same security contract for coturn as we do ret's db access. both are internal servers accessing the db. the fine grained security policy we have is postgrest, which is a SQL proxy, so we have to manage it directly w access control whitelisting and a dedicated user.